modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/contract/DayOfWeekJobId.java

```diff
@@ -0,0 +1,27 @@
+package org.motechproject.scheduler.contract;
+
+import org.motechproject.event.MotechEvent;
+
+public class DayOfWeekJobId extends JobId {
```

Comment:
if we add this class, we should start using it universally. From what I recall day of week jobs are cron jobs underneath and use cron job ids, we can go with those for the time being.